### PR TITLE
Even more derives

### PR DESCRIPTION
**Pull request by victorporof**
Only relevant commit is ee18719

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -36,11 +36,11 @@ pub mod prelude;
 pub mod types;
 pub mod traits;
 
-use std::os::raw::c_void;
+use std::any::Any;
 pub use types::*;
 
 #[repr(C)]
-#[derive(Debug, PartialEq, Eq, Hash)]
```
Comment (victorporof) [inline src/lib.rs]: I accidentally removed these, but I added them back in the "more derives" PR.

**Comment by bschwind** [comment]
@victorporof could you rebase these changes on the most recent master? I want to make sure I don't accidentally get any unintended changes from the previous PR. Thanks!

**Comment by victorporof** [comment]
Rebased.

**Comment by bschwind** [comment]
Thanks for the rebase!

**Pull request merged**